### ISSUE TITLE
BugStomachs with missing taxa code

### ISSUE THREAD

**Issue by philipbaileynar** [namc-utah/namc-bug-db#39]
During a discussion with @Feldspar5 and @courtwrightj01 on 3rd Dec 2020 we discussed bringing the data over from `PilotDB.BugStomachs` to the new system.

There are a total of 2,640 records in `PilotDB.BugStomachs` but 1,694 of them have NULL taxa codes and instead the species of fish stored in the `Species` column. The following are the 10 species that do not have a taxa code. I can't find mention of them in the Taxonomy table `Species` or `CommonNameX` columns (using a simple `LIKE '%Dace%'` etc).

| Species |
| :--- |
| Sculpin |
| Carp |
| Brown trout |
| NULL |
| Mountain whitefish |
| Kendall Warm Springs Dace |
| Rainbow trout |
| Cutthroat trout |
| KOKANEE |
| Utah sucker |

```sql
SELECT Species FROM BugStomachs WHERE Code IS NULL GROUP BY Species;
```

1. Can someone provide taxa codes for these species please? 
2. What should I do with the 203 records that have no species associated with them?

```sql
SELECT Count(*) FROM BugStomachs WHERE code is null and species is null;
```

**Comment by David-Fowler** [comment]
I will get you the updated tables with the included fish codes. From there matches on scientific names (with some sensitization on the side of the bugstomachs table) will give the necessary codes

**Comment by Feldspar5** [comment]
﻿David – It sounds like you are taking care of this fishy issue.  If not, please let me know.

Thanks

Trip

**Comment by David-Fowler** [comment]
Yes, the codes are with taxonomy. Though NULL fish codes will have to be acceptable. The NULL records are still valid as we still have other characteristics of the fish (i.e. length , weight) and no matter how much we would like it, always getting that info from the customer has not been possible to date.

**Comment by courtwrightj01** [comment]
Trip, I think this is green river data that we accidentally did not collect species data for. Can we confirm we don’t have this data on paper data sheets and if not is this data worth anything.
